DQMOffline/Trigger/plugins/ObjMonitor.cc

```diff
@@ -32,9 +33,11 @@ ObjMonitor::ObjMonitor( const edm::ParameterSet& iConfig ) :
   , htjetSelection_ ( iConfig.getParameter<std::string>("htjetSelection"))
   , eleSelection_ ( iConfig.getParameter<std::string>("eleSelection") )
   , muoSelection_ ( iConfig.getParameter<std::string>("muoSelection") )
+  , phoSelection_ ( iConfig.getParameter<std::string>("phoSelection") )
   , njets_      ( iConfig.getParameter<int>("njets" )      )
   , nelectrons_ ( iConfig.getParameter<int>("nelectrons" ) )
   , nmuons_     ( iConfig.getParameter<int>("nmuons" )     )
+  , nphotons_   ( iConfig.getParameter<int>("nphotons")    )
```

Comment:
hi @yiiyama - please make this an unsigned int (the others should be too, but they are already there...)- this will save several int() below..

Reply:
Hi @davidlange6 yes I wanted to make as little change as possible to the original code, but personally I actually feel sick when variables that are semantically unsigned are declared / cast as int..
Taking the opportunity, I changed all n*_ member variables to unsigned. But there would be too much change if I updated the parameter type (to getParameter<unsigned>), so I left that part as int.